recipes/lua-5.1/build.sh

```diff
@@ -0,0 +1,9 @@
+if [ `uname` == Darwin ]; then
+	make macosx INSTALL_TOP=$PREFIX
+fi
+if [ `uname` == Linux ]; then
+	make linux INSTALL_TOP=$PREFIX MYCFLAGS="-I$PREFIX/include -L$PREFIX/lib -DLUA_USE_LINUX -DLUA_USER_DEFAULT_PATH='\"$PREFIX/\"'" MYLDFLAGS="-L$PREFIX/lib -Wl,-rpath=$PREFIX/lib" #  "
+fi
+make install INSTALL_TOP=$PREFIX
+
+
```

Comment:
Can you tie both conditions in a if/else clause and eliminate the extra lines at the bottom. Just to make the file look pretty :wink: